apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -203,6 +209,21 @@ router.post('/', function (req, res, next) {
         }
       });
     });
+  } else if (req.body.__action === 'unsafe_sharing_set_add') {
+    debug('Add question to sharing set');
+    sqldb.queryZeroOrOneRow(
+      sql.sharing_set_add,
+      {
+        course_id: res.locals.course.id,
+        question_id: res.locals.question.id,
+        sharing_set_id: req.body.sharing_set_id,
+      },
+      (err, _result) => {
```

Comment:
You can just omit this entirely.

```suggestion
      (err) => {
```

Reply:
done